docs/markdown/Dependencies.md

```diff
@@ -653,12 +653,19 @@ or as an OSX framework.
 
 *(added 0.51.0)*
 
-Shaderc currently does not ship with any means of detection.
-Nevertheless, Meson can try to detect it using `pkg-config`, but will
-default to looking for the appropriate library manually. If the
-`static` keyword argument is `true`, `shaderc_combined` is preferred.
-Otherwise, `shaderc_shared` is preferred. Note that it is not possible
-to obtain the shaderc version using this method.
+Meson will first attempt to find shaderc using `pkg-config`. Upstream
+currently ships three different `pkg-config` files and by default will
+check them in this order: `shaderc.pc`, `shaderc_combined.pc`, and
+`shaderc_static.pc`. If the `static` keyword argument is `true`, then
+Meson instead checks in this order: `shaderc_combined`, `shaderc_static`,
+and `shaderc.pc`.
+
+If no `pkg-config` file is found, then Meson will try to detect the
+library manually. In this case, it will try to link against `-lshaderc_shared`
+first and then try `-lshaderc_combined`. If the `static` keyword argument
+is `true`, then `-lshaderc_combined` is preferred followed by checking
+`-lshaderc_shared`. Note that it is not possible to obtain the shaderc
```

Comment:
I wonder if it might be clearer to emulate the previous docs, e.g.

> In this case, it will try to link against either `-lshaderc_shared` or `-lshaderc_combined`, preferring the latter if the `static` keyword argument is `true`.

Just trying to find some wording that involves less "enumerate the same things again, but in reverse order". Well-worded documentation is hard...

Reply:
That sounds better to me too.

Reply:
I believe you still want to mention "Note that it is not possible to obtain the shaderc version with this method."

Reply:
crap

Reply:
I mean, it could be worse. You could have done a long and involved rebase of a heavy refactoring series which accidentally dropped actual features. We have been there and done that.

Reply:
The sentence is now undeleted.